docs/source/duckdb_cli_select.md

```diff
@@ -0,0 +1,155 @@
+# Query public, gated and private datasets
+
+Querying datasets is a fundamental step in data analysis. Here, we'll guide you through querying datasets using various methods.
+
+You can query Hugging Face [autoconverted parquet files](https://huggingface.co/docs/datasets-server/en/parquet#conversion-to-parquet) in the `refs/converts/parquet` branch by using the following syntax:
+
+
+```plaintext
+hf://datasets/{my-username}/{my-dataset}@~parquet/{path_to_parquet_file} 
+```
+
+There are [different ways](https://duckdb.org/docs/data/parquet/overview.html) to select your data.
+
+Using `FROM` syntax:
+```bash
+FROM 'hf://datasets/jamescalam/world-cities-geo@~parquet/default/train/0000.parquet' SELECT city, country, region LIMIT 3;
```

Comment:
same here, you can start by showing that people can use jsonl
```
hf://datasets/jamescalam/world-cities-geo/train.jsonl
```

Reply:
Done